libs/wingsdk/src/std/json.ts

```diff
@@ -109,12 +109,12 @@ export class Json {
   /**
    * Try to parse a string into a Json
    *
-   * @macro ((args) => { try { return JSON.parse(args); } catch (err) { return undefined; } })($args$)
+   * @macro ((args) => { try { return args && JSON.parse(args); } catch (err) { return undefined; } })($args$)
```

Comment:
if you'll put here `false`, it'll return `false` instead of `nil`, the same with `0`, I think

Reply:
Since the signature is `str?` it won't be possible to pass these values from wing.

Reply:
and what about passing `""` ? it'll return `""`

Reply:
Nice catch. It should return `undefined` in that case. Checking.

Reply:
Fixed.